src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -759,9 +759,13 @@ export function getEnrichmentColumns(
                 </span>
             ),
             sortBy: (d: ExpressionEnrichmentRow) =>
-                d.groupsSet[group.name].meanExpression,
+                d.groupsSet[group.name]
+                    ? d.groupsSet[group.name].meanExpression
+                    : null,
             download: (d: ExpressionEnrichmentRow) =>
-                d.groupsSet[group.name].meanExpression.toFixed(2),
+                d.groupsSet[group.name]
+                    ? d.groupsSet[group.name].meanExpression.toFixed(2)
+                    : 'NA',
```

Comment:
is there an `NA` constant?

Reply:
Sure, we already has a constant that I can use, updated.